modules/deploying-a-model-from-the-model-catalog.adoc

```diff
@@ -34,7 +34,7 @@ endif::[]
 . From the {productname-short} dashboard, click *AI hub* -> *Catalog*.
 . In the drop-down list, select from the available catalog sources that have been configured by your administrator. The *Default Catalog* is displayed by default. 
 +
-NOTE: OpenShift cluster administrators can configure additional model catalog sources. For more details, see the Kubeflow Model Registry community documentation on link:https://github.com/kubeflow/model-registry/tree/main/manifests/kustomize/options/catalog#configuring-catalog-sources[configuring catalog sources]. 
```

Comment:
Curious about this one. Did the link checker report this as broken?  

Link is working on https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.25/html/working_with_the_model_catalog/deploying-a-model-from-the-model-catalog_working-model-catalog

Reply:
OK, good to know, thanks :)